gadget/update.go

```diff
@@ -20,8 +20,102 @@ package gadget
 
 import (
 	"errors"
+	"fmt"
 )
 
 var (
 	ErrNoUpdate = errors.New("no update needed")
 )
+
+func resolveVolume(old *Info, new *Info) (oldVol, newVol *Volume, err error) {
```

Comment:
This is defined and test but not used in this patch. I'm assuming it's preparation for a future change.

Reply:
Yup, the next batch is already waiting for a PR. I have to control the dosage otherwise nobody would do the reviews :smile: